kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -380,6 +380,6 @@ log_level -i "ARM template generated at $PWD/_output/$MASTER_DNS_PREFIX director
 cd $PWD/_output/$MASTER_DNS_PREFIX 
 
 log_level -i "Deploy the template."
-az group deployment create -g $RESOURCE_GROUP_NAME --template-file azuredeploy.json --parameters azuredeploy.parameters.json > /dev/null
+az group deployment create -g $RESOURCE_GROUP_NAME --template-file azuredeploy.json --parameters azuredeploy.parameters.json --output none
```

Comment:
I think that our documentation is pointing people to check this log file for troubleshooting purposes.
Maybe it is worthwhile to delay this change.

Reply:
None format will only suppress the return success message and not errors and warnings.
The details are mentioned below:
https://docs.microsoft.com/en-us/cli/azure/format-output-azure-cli?view=azure-cli-latest